src/combinatorics/simple_graph/regularity/uniform.lean

```diff
@@ -114,7 +196,12 @@ begin
   exact G.is_uniform_singleton hε,
 end
 
-lemma bot_is_uniform (hε : 0 < ε) : (⊥ : finpartition s).is_uniform G ε :=
+/-- A finpartition is `ε`-uniform (aka `ε`-regular) iff at most a proportion of `ε` of its pairs of
+parts are not `ε-uniform`. -/
```

Comment:
/-- A finpartition of a graph's vertex set is `ε`-uniform (aka `ε`-regular) iff at most a proportion of `ε` of
its pairs of parts are not `ε-uniform`. -/

Reply:
```suggestion
/-- A finpartition of a graph's vertex set is `ε`-uniform (aka `ε`-regular) iff at most a proportion
of `ε` of its pairs of parts are not `ε-uniform`. -/
```